install/install.class.php

```diff
@@ -607,6 +607,8 @@ protected function createInitialConfig() {
          'mqtt_broker_internal_address'    => '127.0.0.1',
          'mqtt_broker_port'                => '1883',
          'mqtt_broker_tls_port'            => '8883',
+         'mqtt_broker_port_backend'        => '1884',
+         'mqtt_broker_tls_port_backend'    => '8884',
```

Comment:
I suggest the defaults for t hose 2 settings be the same as the existing ones : 1883 and 8883.

For most configurations they will be the same. Different values may be required for some very specific cases.